src/main/java/net/fabricmc/tinyremapper/api/ClassHeader.java

```diff
@@ -0,0 +1,30 @@
+package net.fabricmc.tinyremapper.api;
+
+import java.util.List;
+
+public interface ClassHeader {
+	Classpath getClasspath();
+
+	int getAccess();
+
+	String getName();
+
+	String getSuperName();
+
+	String getSignature();
+
+	List<String> getInterfaceList();
```

Comment:
getInterfaces? Why is this a List and the other is an Iterable?

Reply:
Order is important here but not for members (I think)